lib/code_corps_web/controllers/user_controller.ex

```diff
@@ -20,6 +21,7 @@ defmodule CodeCorpsWeb.UserController do
       |> Query.id_filter(params)
       |> Query.limit_filter(params)
       |> Query.user_filter(params)
+      |> Users.project_filter(params)
```

Comment:
I think it's probably better if we just alias `CodeCorps.Accounts` and use the more explicit `Accounts.Users` here. What do you think?

Reply:
Thats a good idea, its more explicit and will cause less confusion later.